worker/src/main/java/gov/cms/ab2d/worker/processor/JobPreProcessorImpl.java

```diff
@@ -72,6 +73,20 @@ public Job preprocess(String jobUuid) {
         if (contract == null) {
             throw new IllegalArgumentException("A job must always have a contract.");
         }
+
```

Comment:
add here comments about the ticket number and what we are trying to achieve

Reply:
Will do before merge